votes/0192-binshi-bing-as-pd-reviewer.md

```diff
@@ -0,0 +1,17 @@
+# A Vote for binshi-bing as TiKV Reviewer
+
+## Proposal
+
+[@binshi-bing](https://github.com/binshi-bing) has continuously contributed to `tikv/pd` on different components, including but not limited to storage, raftstore and related components. The main contributions are listed in the following:
```

Comment:
The description seems that not that matched.

Reply:
fixed